pino.js

```diff
@@ -130,6 +132,11 @@ function Pino (level, stream, serializers, stringify, end, name, slowtime, chind
   this._baseLog = flatstr(baseLog +
     (this.name === undefined ? '' : '"name":' + stringify(this.name) + ','))
 
+  if (this.timestamp === false) {
+    this.time = getNoTime
+    return
```

Comment:
If `timestamp` is `false` then there is no reason to check for the `slowtime` option. This `return` short circuits that check.